specification/schemas/payment-providers/PaymentProviderResponse.json

```diff
@@ -19,7 +19,14 @@
             "logo_url",
             "connects_through_oauth2",
             "created_at"
-          ]
+          ],
+          "properties": {
+            "connected": {
```

Comment:
Let's put this property in the `meta`, since it's not information about the payment provider itself, but rather side information based on request parameters.